apps/site/components/withBreadcrumbs.tsx

```diff
@@ -62,7 +62,7 @@ const WithBreadcrumbs: FC<WithBreadcrumbsProps> = ({ navKeys = [] }) => {
 
   return (
     <Breadcrumbs
-      links={getBreadrumbs()}
+      links={getBreadCrumbs()}
```

Comment:
nit: "Breadcrumbs" is one word, so it shouldn't be camelCased.

Reply:
oops... yeah good point, let me fix that :slightly_smiling_face:

Reply:
Fixed, thanks :slightly_smiling_face: